test/functionalTest/cases/0967_v2_query_op/illegal_chars_in_url_param.test

```diff
@@ -79,20 +77,6 @@ echo
 echo
 
 
```

Comment:
Maybe the deleted tests should be "recovered" once we have adjust the "semicolon relaxion" to only coords= and geometry= ?